frontend/src/components/filter/trainingReportFilters.js

```diff
@@ -32,6 +35,13 @@ const defaultDateValues = {
   'is on or before': '',
 };
 
+const handleArrayQuery = (q) => {

```

Comment:
would suggest using

```js
import { handleArrayQuery } from './helpers';
```

Just converted all the other filters over on my PR #1831

Reply:
updated to use helper.